srcpkgs/python3-engineio/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-engineio'
+pkgname=python3-engineio
+version=4.3.0
+revision=1
+wrksrc="python-engineio-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-Sphinx"
+makedepends="${depends}"
```

Comment:
$depends is not defined here and you probably don't want the dependencies in makedepends